Cargo.toml

```diff
@@ -16,6 +16,7 @@ rand_core = "0.6.0"
 defmt = { version = "0.3", optional = true }
 log = { version = "0.4.14", optional = true }
 embedded-io = { version = "0.4.0", features = ["async"]}
+embedded-io-async = { version = "0.6.0", features = ["defmt-03"]}
```

Comment:
I would suggest not using defmt-03 by default.

Reply:
Apparently that's how the defmt feature is called :) 
https://github.com/rust-embedded/embedded-hal/tree/master/embedded-io-async

Reply:
@lulf Is this resolved ?